templates/taxi/driver_list.html

```diff
@@ -7,7 +7,7 @@ <h1>Driver list</h1>
       {% for driver in driver_list%}
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          ({{ driver.first_name }} {{ driver.last_name }}) {% if request.user == driver %}(Me){% endif %}
```

Comment:
The template correctly implements the requirement to add '(Me)' next to the current user's name in the driver list, as specified in the task: 'In Driver's list view add (Me), if this is a current user.'